gamma:          100
contrast:       10
sharpness:      90
brightness:     1
backlight_comp: 2
auto_exposure:  8
exposure:       50
hue:            0
saturation:     0
pwr_line_freq:  1
auto_exp_prio:  0
white_bal_auto: 1
white_bal:      4600

